docs/INSTALL.md

```diff
@@ -79,61 +109,39 @@ export PATH="$PATH:/home/plume/.cargo/bin:/home/plume/.local/bin:/usr/local/sbin
 
 On **Windows**, you'll need, if you don't already have them, to download and install the [Visual C++ 2015 Build Tools](https://www.microsoft.com/en-us/download/details.aspx?id=48159). Then, download the [rustup installer](https://www.rust-lang.org/en-US/install.html) and run it.
 
-## Getting and compiling the Plume source code
+## Getting Plume's source code
 
-Plume needs to be compiled from source.
+Plume needs to be compiled from source. To download the code, run:
 
 ```bash
 git clone https://github.com/Plume-org/Plume.git
 cd Plume
-
-# This may take some time as RustUp will download all
-# the required Rust components, and Cargo will download
-# and compile all dependencies.
-cargo build
```

Comment:
where's the build step gone to?

Reply:
In `cargo install`, which installs `plume` and `plm` on the system.